version4.go

```diff
@@ -4,12 +4,17 @@
 
 package uuid
 
-import "io"
+import (
+	"crypto/rand"
+	"io"
+)
 
 // New creates a new random UUID or panics.  New is equivalent to
 // the expression
 //
 //    uuid.Must(uuid.NewRandom())
+//
+// Deprecated: Use *Version4.NewUUID() instead.
```

Comment:
I especially object to this and the deprecation of `NewRandom` etc. - if the library is going to provide ctor sugar, it should at least provide the most common one as the shortest!

Reply:
I mean, "deprecation" ultimately means little in the face of semantic versioning, unless and until v2 of this library rolls around. I can alter the comment to instead suggest considering using `*Version4.NewUUID()` if that would make you feel better.

Reply:
This is not deprecated.  New and NewString are the two most common ways to create a UUID.  Version 4 is by far the most common type of UUID due to the issues associated with Version 1 (which are documented in the RFC).